core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -60,33 +58,95 @@
       </div>
 
       <!-- Collection changes to be commited -->
-      <h3><span ng-if="getChangeListCount() > 0"> <em>(<[getChangeListCount()]> changes pending)</em></h3>
+      <h3>
+        <span ng-if="getChangeListCount() > 0">
+          <em>(<[getChangeListCount()]> changes pending)</em>
+      </h3>
+
       <br>
-      <input ng-model="commitMessage" type="text" placeholder="Commit Message"><button ng-click="saveCollection(commitMessage)" ng-disabled="validationIssues.length > 0">Save Collection</button>
-      <button ng-click="publishCollection()" ng-if="isPrivate">Publish Collection</button>
-      <button ng-click="unpublishCollection()" ng-if="!isPrivate && canUnpublish">Unpublish Collection</button>
+      <input
```

Comment:
I thought it was an interesting approach; wanted to get your thoughts on it. I reverted back to the conventional formatting.